RadarSDK/Radar.m

```diff
@@ -406,6 +407,18 @@ + (void)sendLogConversionRequestWithName:(NSString * _Nonnull) name
     }];
 }
 
++ (void)logOpenedAppConversion {
+    // if opened_app has been logged in the last 1000 milliseconds, don't log it again
+    NSTimeInterval lastAppOpenTimeInterval =[[NSDate date] timeIntervalSinceDate:[RadarSettings lastAppOpenTime]];
+    if (lastAppOpenTimeInterval * 1000 > 1000) {
```

Comment:
if wea re multiplying the left by 1k why don't we just do `if (lastAppOpenTimeInterval > 1)`